security/rbac/docs/index.rst

```diff
@@ -0,0 +1,460 @@
+      confluent iam rolebinding create --principal User:${USER_ADMIN_KSQL} --role ResourceOwner --resource Topic:${CSAS_STREAM1} --kafka-cluster-id $KAFKA_CLUSTER_ID
+      confluent iam rolebinding create --principal User:${USER_KSQL} --role ResourceOwner --resource Topic:${CTAS_TABLE1} --kafka-cluster-id $KAFKA_CLUSTER_ID
+      confluent iam rolebinding create --principal User:${USER_ADMIN_KSQL} --role ResourceOwner --resource Topic:${CTAS_TABLE1} --kafka-cluster-id $KAFKA_CLUSTER_ID
+      confluent iam rolebinding create --principal User:${USER_ADMIN_KSQL} --role ResourceOwner --resource Topic:_confluent-ksql-${KSQL_SERVICE_ID} --prefix --kafka-cluster-id $KAFKA_CLUSTER_ID
+
+Control Center
+~~~~~~~~~~~~~~
+
+- :devx-examples:`Additional RBAC configurations required for control-center-dev.properties|security/rbac/delta_configs/control-center-dev.properties.delta`
+-  Role bindings:
+
+   .. code:: bash
+
+      # Control Center Admin
+      confluent iam rolebinding create --principal User:$USER_ADMIN_C3 --role SystemAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID
+   
+      # Control Center user
+      confluent iam rolebinding create --principal User:$USER_CLIENT_C --role DeveloperRead --resource Topic:$TOPIC1 --kafka-cluster-id $KAFKA_CLUSTER_ID
+      confluent iam rolebinding create --principal User:$USER_CLIENT_C --role DeveloperRead --resource Topic:$TOPIC2_AVRO --kafka-cluster-id $KAFKA_CLUSTER_ID
+      confluent iam rolebinding create --principal User:$USER_CLIENT_C --role DeveloperRead --resource Subject:${TOPIC2_AVRO}-value --kafka-cluster-id $KAFKA_CLUSTER_ID --schema-registry-cluster-id $SCHEMA_REGISTRY_CLUSTER_ID
+      confluent iam rolebinding create --principal User:$USER_ADMIN_C3 --role ClusterAdmin --kafka-cluster-id $KAFKA_CLUSTER_ID --schema-registry-cluster-id $SCHEMA_REGISTRY_CLUSTER_ID
+      confluent iam rolebinding create --principal User:$USER_CLIENT_C --role DeveloperRead --resource Connector:$CONNECTOR_NAME --kafka-cluster-id $KAFKA_CLUSTER_ID --connect-cluster-id $CONNECT_CLUSTER_ID
+   
+General Rolebinding Syntax
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+General rolebinding syntax:
+
+.. code:: bash
+
+   confluent iam rolebinding create --role [role name] --principal User:[username] --resource [resource type]:[resource name] --[cluster type]-cluster-id [insert cluster id] 
+
+Available role types and permissions can be found :ref:`here <rbac-predefined-roles>`.
+
+Resource types include: Cluster, Group, Subject, Connector, TransactionalId, Topic.
+
+
+Listing a Users roles
+~~~~~~~~~~~~~~~~~~~~~
+
+General listing syntax:
+
+.. code:: bash
+
+   confluent iam rolebinding list User:[username] [clusters and resources you want to view their roles on]
+
+For example, list the roles of ``User:bender`` on Kafka cluster ``KAFKA_CLUSTER_ID``
+
+.. code:: bash
+
+   confluent iam rolebinding list --principal User:bender --kafka-cluster-id $KAFKA_CLUSTER_ID 
+
+
+.. _rbac_demo_docker:
+
+Docker
```

Comment:
need to make this title more descriptive, e.g. `Demo 2: Install RBAC using Docker`

Reply:
Ack